tests/js/common/shell/shell-validation-rocksdb.js

```diff
@@ -360,6 +360,28 @@ function ValidationBasicsSuite () {
       }
 
     },
+    // AQL  ////////////////////////////////////////////////////////////////////////////////////////////
+    testGET_SCHEMA: () => {
+      const v =  validatorJson;
+      v.level = "strict";
+      testCollection.properties({"validation" : v });
+      sleepInCluster();
+
+      let res = db._query(`RETURN GET_SCHEMA("${testCollectionName}")`).toArray();
+      assertEqual(res[0], v.rule);
```

Comment:
We should also have tests for getting the schema of a non-existing collection (which should throw an error), and for a collection that does not have any schema (which should return `null`).